src/audio/codec_adapter/codec/generic.c

```diff
@@ -261,7 +277,12 @@ int codec_process(struct comp_dev *dev)
 		return -EPERM;
 	}
 
-	ret = codec->ops->process(dev);
+	// TODO: how to set cd->passthrough flag?
+	if (cd->passthrough) {
```

Comment:
via topology?

Reply:
Yea, but we have no example of enum or on/off controls I think is why it's left as a todo

Reply:
@juimonen any example so thi scan be added into topology and code here ?

Reply:
we have examples of enums in tdfb, @johnylin76 please us those